packages/cozy-client/src/models/document/locales/fr.json

```diff
@@ -54,7 +54,7 @@
       "gradebook": "Bulletin de notes",
       "student_card": "Carte d'étudiant",
       "school_attendance_certificate": "Certificat de scolarité",
-      "resume": "CV",
+      "resume": "Curriculum Vitae",
```

Comment:
Who in the world say "Curriculum Vitae"? :smile: 
It's obviously not a big deal, but what is the reason for this change? For me it's just longer and even less clear (I'm pretty sure some people don't even know what a "Curriculum Vitae" actually is).

Reply:
Ca fait partie d'un retour client, je laisse @BenjaminMty développer si besoin :)

Reply:
yes it's a client request, apparently CV is less clear than Curriculum vitae but in English we keep the word resume.

Reply:
Suite à nos échanges interne, on annule ce changement.